src/snapred/backend/data/DataFactoryService.py

```diff
@@ -128,6 +128,10 @@ def getNormalizationDataWorkspace(self, runId: str, useLiteMode: bool, version:
         path = self.getNormalizationDataPath(runId, useLiteMode, version)
         return self.groceryService.fetchWorkspace(os.path.join(path, name) + ".nxs", name)
 
+    @validate_arguments
+    def getNormalizationVersion(self, runId: str, useLiteMode: bool):
+        return self.lookupService._getVersionFromNormalizationIndex(runId, useLiteMode)
```

Comment:
could you make a wrapper method in localdataservice called `getVersionFromNormalizationIndex` that just invokes `_getVersionFromNormalizationIndex`, typically methods prepended with an '_' are considered private.